lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1001,6 +1001,37 @@ static Optional<bool> emitSIBAfterSILGen(CompilerInvocation &Invocation,
                       Instance.getASTContext(), MSF);
 }
 
+/// Perform "stable" optimizations that are invariant across compiler versions.
+static bool performStableOptimizations(CompilerInvocation &Invocation,
```

Comment:
You're not to blame for this, but we've moved to calling these the "mandatory SIL passes" rather than "stable optimizations".

Reply:
Name changed: "performMandatorySILPasses" but comment left alone.